llpc/tool/llpcCompilationUtils.cpp

```diff
@@ -222,11 +236,8 @@ Expected<BinaryData> compileGlsl(const std::string &inFilename, ShaderStage *sta
   void *bin = new char[binSize]();
   llvm::copy(llvm::make_range(spvBin, spvBin + binSize / sizeof(unsigned)), reinterpret_cast<unsigned *>(bin));
 
-  textSize = binSize * 10 + 1024;
-  std::vector<char> spvText(textSize, '\0');
```

Comment:
I note that this version here initializes spvText to '\0' whereas the refactored code (and the other instances in the old code) do not - is the new version safe?